docs/source/en/_toctree.yml

```diff
@@ -19,6 +19,8 @@
     title: Train a diffusion model
   - local: tutorials/using_peft_for_inference
     title: Inference with PEFT
+  - local: tutorials/custom_pipelines_components
+    title: Working with fully custom pipelines and components
```

Comment:
Nice points, Steven!

I think it caters to advanced users. So, I would prefer it to be a guide. In that case, where it could live?

Reply:
I think it can live in the [Load community pipelines](https://huggingface.co/docs/diffusers/main/en/using-diffusers/custom_pipeline_overview) page, and then we restructure it to:

```md
# Load community pipelines and components (update the toctree to this title as well)

## Community pipelines

## Community components
```